### PR TITLE
image registry update

### PR DESCRIPTION
**Pull request by kundan2707**
Fixes: #140 
updated registry location in example folder

### CODE REVIEW

**Comment by kundan2707** [comment]
@bowei @thockin  Can you PTAL?

File: examples/ladder-defaultparams.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ spec:
         app: autoscaler
     spec:
       containers:
-        - image: gcr.io/google_containers/cluster-proportional-autoscaler-amd64:{LATEST_RELEASE}
+        - image: registry.k8s.io/cpa/cluster-proportional-autoscaler-amd64:{LATEST_RELEASE}
```
Comment (thockin) [inline examples/ladder-defaultparams.yaml:54]: shouldn't we also drop the "-amd" suffix?

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

File: examples/ladder-defaultparams.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ spec:
         app: autoscaler
     spec:
       containers:
-        - image: gcr.io/google_containers/cluster-proportional-autoscaler-amd64:{LATEST_RELEASE}
```
Comment (BenTheElder) [inline examples/ladder-defaultparams.yaml:54]: are we even publishing these images? google_containers is _two_ registries and multiple years ago

**Pull request merged**